pkgs/by-name/pa/papers/package.nix

```diff
@@ -37,16 +37,16 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "papers";
-  version = "45.0-unstable-2024-03-27";
+  version = "46.1";
```

Comment:
46.2 was released (only metadata changes): https://gitlab.gnome.org/GNOME/Incubator/papers/-/compare/46.1...46.2

Reply:
Updated to 46.2